frontend/src/Frontend/UI/Wallet.hs

```diff
@@ -144,9 +156,11 @@ uiKeyItems model = do
 
     el "tbody" $ do
       events <- listWithKey keyMap (uiKeyItem model)
-      dyn_ $ ffor keyMap $ \keys -> when (Map.null keys) $
-        elClass "tr" "wallet__table-row" $ elAttr "td" ("colspan" =: "6" <> "class" =: "wallet__table-cell") $
-          text "No accounts ..."
+      let selectedNetwork = model ^. network_selectedNetwork
+      dyn_ $ ffor2 keyMap selectedNetwork $ \keys net ->
+        unless (any (isJust . activeAccountOnNetwork net) $ Map.elems keys) $
```

Comment:
Prior to this, if you had no account on the current network but had other accounts (or deleted accounts on this network) you'd have a blank table but with no "no accounts" message.

Reply:
Shouldn't we feed a filtered version of the map to both the list widget and the message widget instead of doing the check in two places?